CalculateCurrentIssues.cls

```diff
@@ -1,71 +1,71 @@
 Global class CalculateCurrentIssues implements Schedulable{
 
-global void execute(SchedulableContext SC) {
+global void execute(SchedulableContext sc) {
      ClearAllIssueStatuses();
      SetNewIssueStatuses();
   }
 
 public static void ClearAllIssueStatuses() {
```

Comment:
Methods must use the `lowerCamelCase` naming convention. Please rename accordingly.